example/src/App.js

```diff
@@ -89,6 +93,47 @@ function App() {
     }
   }, [location]);
 
+  const menu = !isRestMode ? (
+    <Menu
+      id="app-nav-menu"
+      anchorEl={anchorEl}
+      keepMounted
+      open={Boolean(anchorEl)}
+      onClose={handleAppNavClose}
+    >
+      <MenuItem
+        onClick={handleAppNavClose}
+        component={RouterLink}
+        to="/catalog"
+      >
+        {`Catalog ${appType}`}
+      </MenuItem>
+      <MenuItem
+        onClick={handleAppNavClose}
+        component={RouterLink}
+        to="/profile"
+      >
+        {`Profile ${appType}`}
+      </MenuItem>
+      <MenuItem
+        onClick={handleAppNavClose}
+        component={RouterLink}
+        to="/component-definition"
+      >
+        {`Component ${appType}`}
+      </MenuItem>
+      <MenuItem
+        onClick={handleAppNavClose}
+        component={RouterLink}
+        to="/system-security-plan"
+      >
+        {`System Security Plan ${appType}`}
+      </MenuItem>
+    </Menu>
+  ) : (
+    <OSCALDrawerSelector open={drawerIsOpen} handleClose={handleAppNavClose} />
+  );
```

Comment:
Hmm... this is a very off-balance ternary operator. Already the complexity seems to have nudged us to extract it to a variable. Perhaps we should then extract the `<Menu>` to its own component? Or maybe the whole thing?

Or maybe it can be solved a bit by just renaming the variable to `navigation` instead of `menu`?

Reply:
Yeah I agree it is gross. I am not against making a `OSCALMenuSelecter` object of some sort. Not sure if that would be the best name for it.